skia-bindings/build_support/binaries.rs

```diff
@@ -120,12 +121,11 @@ pub fn key(repository_short_hash: &str, features: &[impl AsRef<str>], skia_debug
 
 /// Create the download URL for the prebuilt binaries archive.
 pub fn download_url(tag: impl AsRef<str>, key: impl AsRef<str>) -> String {
-    format!(
-        "https://github.com/rust-skia/skia-binaries/releases/download/{}/{}-{}.tar.gz",
-        tag.as_ref(),
-        ARCHIVE_NAME,
-        key.as_ref()
-    )
+    let binding_url = env::var("SKIA_BINARIES_URL")
```

Comment:
One more thing, use `cargo::env_var()` here, so that the build script is called again when the environment variable changes.

Reply:
Done.